__tests__/app.test.js

```diff
@@ -79,3 +80,35 @@ describe("GET /api/articles/:article_id", () => {
 			});
 	});
 });
+
+describe("GET /api/articles", () => {
+	it("status 200: return an array of objects that have a comment_count as an added column", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then(({ body: { articles } }) => {
+				expect(articles).not.toHaveLength(0);
```

Comment:
We could be more specific with the length check here, as we have access to the test data we could simply count how many are in there and be confident that it won't change as we are seeding before each test.